src/Microsoft.AspNet.Server.Kestrel/Infrastructure/MemoryPoolIterator2.cs

```diff
@@ -7,8 +7,10 @@
 
 namespace Microsoft.AspNet.Server.Kestrel.Infrastructure
 {
-    public struct MemoryPoolIterator2
+    public partial struct MemoryPoolIterator2
     {
+        private readonly static int _vectorSpan = Vector<byte>.Count; 
```

Comment:
Not trusting it to be an intrinsic const when its not hardware accelerated as it is going through a [static property to a non-readonly static](https://github.com/dotnet/corefx/blob/master/src/System.Numerics.Vectors/src/System/Numerics/Vector.cs#L52-L59). Whereas know that the `readonly static int` will be a jitted const.